src/analysis/tax_transfer_funcs/benefits/wohngeld.py

```diff
@@ -42,13 +44,21 @@ def calc_wg_rent(df, tb, hhsize):
     This function yields the relevant rent for calculating the wohngeld.
     """
     # There are also min and max values for this.
+    # If 'Mietstufe' is not given, choose '3'.
+    if 'mietstufe' in df.columns:
+        mietstufe = int(df['mietstufe'].max())
+    else:
+        mietstufe = 3
+
+    assert(mietstufe in range(1, 7))
     # First max rent
     if tb["yr"] >= 2009:
```

Comment:
I know this is a leftover from my code. But didn't we agree to implement unified function interfaces and then hand over the functions via a dictionary for now? If we want each different method of calculating things as function on his own, this would be the way to go. Otherwise we could also just check the year in the function and if before 2009 we could set the key to "m".

Reply:
Correct. Will be addressed